tests/integration/CMakeLists.txt

```diff
@@ -19,9 +19,6 @@ target_sources(foobar PRIVATE
     vk_format_utils_2.c # Need two translation units to test if header file behaves correctly.
 )
 
-# NOTE: Because VulkanHeaders is a PUBLIC dependency it needs to be found prior to VulkanUtilityLibraries
-find_package(VulkanHeaders REQUIRED CONFIG)
-
```

Comment:
All of the above installation changes are done for the sake of removing this call for the user of the library.

This will now be handled by `find_package(VulkanUtilityLibraries REQUIRED CONFIG)` below.